malariagen_data/ag3.py

```diff
@@ -314,6 +320,482 @@ def _repr_html_(self):
         """
         return html
 
+    def genome_sequence(self, region, inline_array=True, chunks="native"):
+        """Access the reference genome sequence.
+
+        Parameters
+        ----------
+        region: str or list of str or Region or list of Region
+            Contig name (e.g., "2L"), gene name (e.g., "AGAP007280"), genomic
+            region defined with coordinates (e.g., "2L:44989425-44998059") or a
+            named tuple with genomic location `Region(contig, start, end)`.
+            Multiple values can be provided as a list, in which case data will
+            be concatenated, e.g., ["3R", "3L"].
+        inline_array : bool, optional
+            Passed through to dask.array.from_array().
+        chunks : str, optional
+            If 'auto' let dask decide chunk size. If 'native' use native zarr
+            chunks. Also, can be a target size, e.g., '200 MiB'.
+
+        Returns
+        -------
+        d : dask.array.Array
+            An array of nucleotides giving the reference genome sequence for the
+            given contig.
+
+        """
+        region = self.resolve_region(region)
+
+        if region.contig in self.contigs:
+            d = super().genome_sequence(
+                region, inline_array=inline_array, chunks=chunks
+            )
+            return d
+
+        elif region.contig in ["2RL", "3RL"]:
+            genome = self.open_genome()
+            contigs = (
+                region.contig[0] + region.contig[1],
+                region.contig[0] + region.contig[1],
+            )
+
+            d = da.concatenate(
+                [
+                    da_from_zarr(
+                        genome[contig], inline_array=inline_array, chunks=chunks
+                    )
```

Comment:
```suggestion
                    super().genome_sequence(region=contig, inline_array=inline_array, chunks=chunks)
```

Small nit, but generally nice to keep things tidy by calling superclass method where possible.